docs/terminal/shell-integration.md

```diff
@@ -91,6 +91,35 @@ When using shell integration, it has a "quality" associated with it that declare
 
 To view the shell integration quality, hover the terminal tab. Optionally, select **Show Details** on the hover to view more detailed information.
 
+## IntelliSense (Preview)
```

Comment:
@ntrogh should we move this back to the bottom until it's no longer preview? This is the right place to put it when it's no longer preview imo.

Reply:
Agree. Preview features should go lower down the article.